.nycrc

```diff
@@ -0,0 +1,6 @@
+{
+  "exclude": [
+    "**/internal/process/write-coverage.js"
+  ],
+  "reporter": ["html", "text"]
```

Comment:
Does adding the "text" reporter here affect the default output ?  Just wondering if results on https://coverage.nodejs.org/ will be affected.

Reply:
@mhdawson I don't think it should; `text` will result in a nice summary being written to the terminal, for the user who just ran tests. `html` results in an `index.html` (etc) being written to the `./coverage` folder, which I believe is what is in turn used on `coverage.nodejs.org`.

@addaleax I think I'm correct on this one?

Reply:
Yes, that sounds right to me :)